servers/src/main/java/tachyon/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -41,6 +42,7 @@
 /**
  * This class is a Thrift handler for file system master RPCs invoked by a Tachyon client.
  */
+@ThreadSafe
```

Comment:
if `FileSystemMaster` is not thread safe, then I think we have the same problem as with `BlockMaster` ... I have removed the annotation for now